test/integration/metrics_health_test.lua

```diff
@@ -58,6 +66,29 @@ g.test_metrics_health_handler = function()
     t.assert_equals(resp.status, 200)
 end
 
+g.test_metrics_custom_is_health_handler = function()
+    local main_server = g.cluster:server('main')
+    main_server:exec(function()
+        local metrics = require('cartridge.roles.metrics')
```

Comment:
Replaced.
What is the difference
```
require('cartridge.roles.metrics')
```
and
```
cartridge.service_get('metrics')
```
?

Reply:
`service_get` is by-design way to interact with role enabled on the instance. The difference between `require` and `get_service` is that `require` is always available when the file exist, but `get_service` returns non-nil module only if role is actually enabled. It doesn't that much of an issue in case of a permanent role `cartridge.roles.metrics`, especially in tests, but since it is common to copypaste test contents to applications, it is better to call appropriate API, if possible.